frontend/src/services/AuthService.ts

```diff
@@ -119,8 +121,11 @@ const removeFamUser = () => {
     localStorage.removeItem(CURRENT_SELECTED_APPLICATION_KEY);
 }
 
-const storeFamUser = (famLoginUser: FamLoginUser | null | undefined) => {
+async function storeFamUser(famLoginUser: FamLoginUser | null | undefined) {
```

Comment:
Hi Jota, we refactor all function to write in the format of `const storeFamUser = () => {}`, for async, you can use const storeFamUser = async () => {}`, please keep in mind we have to keep consistency, thanks!!

Reply:
Thank you, I already changed it.